datalad_debian/add_distribution.py

```diff
@@ -0,0 +1,80 @@
+import logging
+
+from datalad.distribution.dataset import (
+    EnsureDataset,
+    datasetmethod,
+    require_dataset,
+)
+from datalad.interface.base import (
+    Interface,
+    build_doc,
+)
+from datalad.interface.utils import (
+    eval_results,
+)
+from datalad.support.constraints import (
+    EnsureNone,
+    EnsureStr,
+)
+from datalad.support.param import Parameter
+
+lgr = logging.getLogger('datalad.debian.add_distribution')
+
+
+@build_doc
+class AddDistribution(Interface):
+    """Create a distribution dataset to an package repository dataset
+    """
+    _params_ = dict(
+        dataset=Parameter(
+            args=("-d", "--dataset"),
+            doc="""specify the package repository dataset to add the
+            distribution to""",
+            constraints=EnsureDataset() | EnsureNone()),
+        source=Parameter(
+            args=("source",),
+            metavar='SOURCE',
+            doc="""URL, DataLad resource identifier, local path or instance of
+            distribution dataset to be added""",
+            constraints=EnsureStr()),
+        name=Parameter(
+            args=('name',),
+            metavar='NAME',
+            doc="""name to add the distribution dataset under""",
```

Comment:
I don't quite parse this. I feel like it's missing the noun of whatever is "named".